app/models/setting.rb

```diff
@@ -1,7 +1,7 @@
 require 'resolv'
 
 class Setting < ApplicationRecord
-  audited :except => [:name, :description, :category, :settings_type, :full_name, :encrypted], :on => [:update]
+  audited :except => [:name, :category], :on => [:update]
```

Comment:
**Food for thought:** we are not yet ready to drop category as some settings still may rely on the column existence to validate values (looking at you Katello!), but is it worth delaying this to Foreman 3.4, when we are saying setting subclassing is deprecated, or will we drop that separately?